crates/sui-e2e-tests/tests/transaction_orchestrator_tests.rs

```diff
@@ -181,13 +183,25 @@ async fn test_transaction_orchestrator_reconfig() {
 
     test_cluster.trigger_reconfiguration().await;
 
-    let epoch = test_cluster.fullnode_handle.sui_node.with(|node| {
-        node.transaction_orchestrator()
-            .unwrap()
-            .quorum_driver()
-            .current_epoch()
-    });
-    assert_eq!(epoch, 1);
+    // After epoch change on a fullnode, there could be a delay before the transaction orchestrator
+    // updates its committee (happens asynchronously after receiving a reconfig message). Use a timeout
+    // to make the test more reliable.
+    timeout(Duration::from_secs(5), async {
```

Comment:
use TestCluster::wait_for_epoch https://github.com/MystenLabs/sui/blob/main/crates/test-utils/src/network.rs#L202?

Reply:
That doesn't guarantee that the transaction orchestrator has also reconfigured. In fact, `trigger_reconfiguration` already calls `wait_for_epoch`.